djangocms_picture/migrations/0012_alter_picture_cmsplugin_ptr.py

```diff
@@ -0,0 +1,20 @@
+# Generated by Django 4.1.8 on 2023-05-04 15:04
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('cms', '__first__'),
```

Comment:
I'd rather this be latest.

Reply:
latest migration from develop branch

Reply:
I've changed code to use latest migration found in the develop branch of django-cms repo.

now the code is the same with #118 

The original idea was to accommodate cases where users update djangocms-picture, but may keep django-cms in older version than does not includes the latest migration. I've chosen the first migration, because that's where `cmsplugin_ptr` is created.

Reply:
I think @marksweb referred to `'__latest__'` instead of `'__first__'`. This would allow you to keep cms outdated (for whatever reason ;-) )

Reply:
Looking back at this, I think it needs to be `__first__` because if if's latest and we release a new migration, the applied migration here would depend on a migration yet to be applied and that causes an error.